index.html

```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
                             </div>
                         </div>
                     </section>
+                    <section class = contact-section> 
+                        <h3 class ="contact-description">
+                            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                        </h3>
+                        <ul class="input-filed">
+                        <li><label for="email"> <input type="email" id="email" name="email" placeholder="Full name"></label></li>
+                       <li> <label for="e-mail">
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>  
+                     </ul>
+                       <textarea name="comment" id="comment" cols="30" rows="10">write me something</textarea>
+                       <button class="getintouch">get in touch<button>
```

Comment:
- [ ] Kindly ensure that your buttons also follow the design guidelines, such as `enabled`, `hover`, `pressed`, and `disabled`, so as to have a pixel-perfect design like you'll soon be doing in your job 🙂.

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/229573539-fc09fa49-7de0-4b4c-b6c7-b497ec95e922.gif)

- Figma design
<img width="274" alt="button2" src="https://user-images.githubusercontent.com/41163571/227492511-ac99583d-add1-41a8-b99f-2b2f12771941.png">

- And please remove the black border around the button, and fix the capitalization of the text in the button.